packages/auth/auth0-react/src/routes/router.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  createBrowserRouter,
+  createRoutesFromElements,
+  Route,
+} from "react-router-dom";
+
+import Root from "../layouts/Root";
+
+import Index from "../pages/Index";
+import About from "../pages/About";
+import Error from "../pages/errors/Error";
+import NotFound from "../pages/errors/NotFound";
+import AdminIndex from "../pages/admin/AdminIndex";
+import AuthenticationGuard from "../components/higherOrder/AuthenticationGuard";
+
+const errorComponent = <Error title="500: Server error" />;
+
+export const router = createBrowserRouter(
+  createRoutesFromElements(
+    <Route path="/" element={<Root />}>
```

Comment:
This is critical for the Auth0 context provider to work correctly. It's wrapped **around** the `<Outlet />` in the Root.tsx file so we can have a one-line router in the index.tsx file.